main.go

```diff
@@ -37,20 +39,28 @@ func (c *Column) ToFieldSchema() *bigquery.FieldSchema {
 	f.Name = c.Name
 	f.Required = c.IsNullable == "NO"
 
+	switch c.Name {
+	// Allow BQ to convert special Epoch columns to timestamp
```

Comment:
@sbramin wait, is this a special column name in your project? Or why is this being done this way based on the column name itself?

What is the actual column type according to `information_schema`?

Reply:
Had forgotten about that.  The columns are stored as integers in pgsql and hold epoch values.  Both pgsql and BQ have a little magic which lets you insert epoch values into timestamp columns and it automatically renders them as correct timestamp values. For my apps I prefer reading/writing in epoch but once it gets to BQ other people are less keen so I take advantage of this auto converting.  

I'll remove this / perhaps make it a flag.